### PR TITLE
Find time points

### PR DESCRIPTION
**Pull request by allison-cy-wu**
Added in a wrapper function for date_period to find multiple time points.
Test is also implemented in _test folder.

### CODE REVIEW

File: utility_functions/date_period.py [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 from utility_functions.benchmark import timer
 from utility_functions.custom_errors import *
 from pytz import timezone
 import pytz
 
 
+def find_time_points(
+        start_date: str,
+        period_list: List[int],
+) -> Tuple[str]:
```
Comment (richard-winkler) [inline utility_functions/date_period.py:13]: Might be a good idea to add a quick docstring to make it clear that period_list should be a list of days to add to the start_date value to get the increased or decreased dates.

**Review by richard-winkler — DISMISSED** [review]

**Review by richard-winkler — DISMISSED** [review]

File: utility_functions/date_period.py (hunk 1 above)
Comment (allison-cy-wu) [reply utility_functions/date_period.py:13]: Added docstring to the function just now!

**Review by allison-cy-wu — COMMENTED** [review]
Added docstring to the function just now!

File: utility_functions/date_period.py [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+    :param start_date: str and needs to be in YYYYMMDD format
+    :param period_list: has to be a list of integers
+    :return: Tuple of dates based on the order of the list input.
+    """
+    time_points = []
+    for p in period_list:
+        time_points.append(date_period(p, start_date)[1])
+
+    return time_points
+
```
Comment (allison-cy-wu) [inline utility_functions/date_period.py:27]: @richard-winkler  added the docstrings now.  Please let me know if it doesn't seem clear to you.  Thank you!

**Review by allison-cy-wu — COMMENTED** [review]
Added test and docstring for find_time_points

**Review by richard-winkler — APPROVED** [review]
Everything looks good to me!

**Pull request merged**